Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Traits.adq

```diff
@@ -2080,13 +2080,8 @@
 				},
 				{
 					"type": "conditional_modifier",
-					"situation": "on all IQ rolls to wake up or to recover from surprise or mental stun",
+					"situation": "on all IQ rolls to wake up and to recover from mental stun",
 					"amount": 6
-				},
-				{
-					"type": "conditional_modifier",
-					"situation": "to initiative rolls for your side (+2 if you are the leader)",
-					"amount": 1
```

Comment:
No initiative concept in DFRPG, either? (can you tell I don't use DFRPG that much?)

Reply:
There is definitely no initiative concept in DFRPG. I'll double-check the other items tomorrow.